requirements.txt

```diff
@@ -111,6 +111,7 @@ pexpect==4.8.0
 pickleshare==0.7.5
 Pillow==8.2.0
 pluggy==1.0.0
+pre-commit==2.16.0
```

Comment:
The `requirements.txt` file will be overwritten and updated automatically, on commit to the develop branch, by `ci-dependencies-linux.yml` in the CI.

Currently, we only include [test,train,camera-obs] options from `setup.py` inside `requirements.txt` file.

If needed, perhaps `pre-commit==2.16.0` and `pylint==2.12.2` can be included in the [dev] option in `setup.py` file.

Reply:
I have made this change, is there anything else?